src/rules/block-no-empty/__tests__/index.js

```diff
@@ -48,3 +48,17 @@ testRule(rule, {
     column: 18,
   } ],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [undefined],
+  skipBasicChecks: true,
+  syntax: "sugarss",
+
+  reject: [{
+    code: ".a",
+    message: messages.rejected,
+    line: 1,
+    column: 3,
```

Comment:
@davidtheclark i think we can get error in other rules where we use `beforeBlockString`. Inside rule i do get `between` if present return `length` if not present `length - 1` inside rule. I think it is normal and we should see to utils for this and add tests to rules where we use `beforeBlockString`.